### DISCUSSION TITLE
Openwrt openclash fake-ip 模式 路由器自身不被代理

### DISCUSSION THREAD

**Discussion by brainiac19** [vernesong/OpenClash#3716] [Q&A]
路由器自身可以获得fake-ip，但无法建立连接。
```
wget www.baidu.com
Resolving www.baidu.com... 198.18.0.8
Connecting to www.baidu.com|198.18.0.8|:80... failed: Operation timed out.
```
已开启自身代理
dns是dnsmasq-openclash-adguardhome-mosdns，虽然我不认为是dns的问题但还是贴在这

Core: Clash.Meta alpha-gcc64297, Client: v0.45.157-246
没看到别人有大量这个情况，不知道原因，故只能在这问一下。

**Comment by ghost** [comment]
为什么不认为是dns问题？

**Comment by brainiac19** [comment]
> 为什么不认为是dns问题？

如果能成功拿到fakeip是dns有问题的表现我真的不知道怎么才叫路由自身代理了。

**Reply by ghost** [reply]
路由本机代理只表示核心会主动接手处理来自本机的连接信息

不知道你理解成什么了

**Reply by brainiac19** [reply]
dns更改为dnsmasq-openclash, 即dnsmasq转发，关闭自定义openclash上游dns。同时关闭自定义规则，开启路由本机代理。完全相同的症状，可以拿到fakeip，tcp握不上手。当然同时局域网内其他设备透明代理完全正常。

**Comment by brainiac19** [comment]
终于去翻了下最不愿意碰的firewall rules，结果是tailscale添加的规则引起的问题。
